test/ext/r_forecast/test_r_univariate_predictor.py

```diff
@@ -36,7 +36,20 @@
 TOLERANCE = 1e-6
 
 
-@pytest.mark.parametrize("method_name", SUPPORTED_UNIVARIATE_METHODS)
+@pytest.mark.parametrize(
+    "method_name",
+    [
+        (
+            method_name
+            if method_name != "fourier.arima.xreg"
+            else pytest.param(
+                "fourier.arima.xreg",
+                marks=pytest.mark.xfail(reason="Known bug"),
+            )
+        )
+        for method_name in SUPPORTED_UNIVARIATE_METHODS
+    ],
+)
```

Comment:
Nit: Maybe the following would be shorter / more readable?
```python
if method_name == "fourier.arima.xreg":
    pytest.xfail("Known bug")
```

`pytest.skip` inside the

Reply:
also that